src/lib/scroll/ScrollElement.js

```diff
@@ -40,13 +32,10 @@ class ScrollElement extends Component {
   
 
   handleWheel = e => {
-    const { traditionalZoom } = this.props
-
-    
+    e.preventDefault()
```

Comment:
This is preventing the horizontal scroll from working. I think this can be fixed with 

```
handleWheel = e => {
    const isHorizontalScrolling = e.wheelDeltaY === 0

    // zoom in the time dimension
    if (e.ctrlKey || e.metaKey || e.altKey) {
      e.preventDefault()
      const parentPosition = getParentPosition(e.currentTarget)
      const xPosition = e.clientX - parentPosition.x

      const speed = e.ctrlKey ? 10 : e.metaKey ? 3 : 1

      // convert vertical zoom to horiziontal
      this.props.onWheelZoom(speed, xPosition, e.deltaY)
    } else if (e.shiftKey) {
      e.preventDefault()
      // shift+scroll event from a touchpad has deltaY property populated; shift+scroll event from a mouse has deltaX
      this.props.onScroll(this.scrollComponent.scrollLeft + (e.deltaY || e.deltaX))
      // no modifier pressed? we prevented the default event, so scroll or zoom as needed
    } else if (isHorizontalScrolling) {
      e.preventDefault()
      this.props.onScroll(this.scrollComponent.scrollLeft + e.deltaX) 
      }
    }

  }
```
Also, this will cause performance issues, so you should consider adding `window.requestAnimationFrame` when calling `this.props.onScroll(...) `

Reply:
Hi @DarioG, please check the pr: https://github.com/namespace-ee/react-calendar-timeline/pull/834 

fixed this issue it was related to the another method